tests/test_convergence.py

```diff
@@ -219,6 +219,15 @@ def create_basis(self, m):
         e = ElementTriCR()
         return InteriorBasis(m, e)
 
+class ConvergenceTriCCR(ConvergenceTriP1):
+
+    rateL2 = 2.0
+    rateH1 = 1.0
```

Comment:
You should verify what L2 and H1 convergence rate to expect from the references. The test seems to suggest that `rateL2` should be 3.0. I can also check it later.

Btw, can you also add similar test `ConvergenceTetCCR`? See `ConvergenceTetCR` for an example.

Reply:
Sure :) !!

Reply:
I have added the convergence rates `rateL2` and `rateH1` for `ElementTetCCR` as well. Though, looking till 4.37 in the paper by Crouzeix and Raviart everything looks fine I will take a fresh look at the derivation tomorrow.